### DISCUSSION TITLE
Why a custom workdir argument instead of the standard working-directory?

### DISCUSSION THREAD

**Discussion by FezVrasta** [docker/bake-action#244] [Q&A] [answered]
Hi, thanks for the action! I was wondering why do you provide a custom workdir argument rather than relying on the standard `working-directory` option every step can use?

**Comment by crazy-max** [comment, marked as answer]
`working-directory` is only available for `run` steps, not actions (`uses`) ones: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory
